frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/PageAccordion.test.tsx

```diff
@@ -36,6 +36,13 @@ jest.mock('./useDeleteLayout', () => ({
   useDeleteLayout: () => mockDeleteFormLayout,
 }));
 
+jest.mock(
+  '../../../../../ux-editor/src/containers/DesignView/PageAccordion/useDeleteLayout',
+  () => ({
+    useDeleteLayout: jest.fn(() => ({ deleteLayout: mockDeleteFormLayout, isPending: false })),
+  }),
+);
```

Comment:
Det er allerede definert en mock for den filen på linjene over. Modifiser heller den i stedet for å overskrive den.